From 4a8f6b0d559f0428e8dbe5b9ea29585c7eeaf059 Mon Sep 17 00:00:00 2001
From: Aihua Xu <aihuaxu@apache.org>
Date: Thu, 19 May 2016 11:47:13 -0400
Subject: [PATCH 0775/1164] CDH-45511 HIVE-13786: Fix the unit test failure
 org.apache.hive.service.cli.session.TestHiveSessionImpl.testLeakOperationHandle
 (Reviewed by Ashutosh Chauhan)

(cherry picked from commit 64454faa3bbc2c1d709779762d9f93a15a1b8c15)

Change-Id: I7488836e7a91d97aec0d2139d8ce6e31bad4e2dd
---
 .../service/cli/session/TestHiveSessionImpl.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/itests/hive-unit/src/test/java/org/apache/hive/service/cli/session/TestHiveSessionImpl.java b/itests/hive-unit/src/test/java/org/apache/hive/service/cli/session/TestHiveSessionImpl.java
index cb13796..257ea24 100644
--- a/itests/hive-unit/src/test/java/org/apache/hive/service/cli/session/TestHiveSessionImpl.java
+++ b/itests/hive-unit/src/test/java/org/apache/hive/service/cli/session/TestHiveSessionImpl.java
@@ -70,7 +70,7 @@ protected synchronized void release(boolean userAccess) {
     Map<String, String> confOverlay = new HashMap<String, String>();
     String hql = "drop table if exists table_not_exists";
     Mockito.when(operationManager.newExecuteStatementOperation(same(session), eq(hql),
-        (Map<String, String>)Mockito.any(), eq(true), eq(0))).thenReturn(operation);
+        (Map<String, String>)Mockito.any(), eq(true), eq(0L))).thenReturn(operation);
 
     try {
 
-- 
1.7.9.5

